apps/fhir/bluebutton/views/home.py

```diff
@@ -1,5 +1,6 @@
 import json
-import logging
+
+import apps.logging.request_logger as logging
```

Comment:
These functions have access to the `request` so we should take advantage and pass it in to the logger